plume-front/src/editor.rs

```diff
@@ -80,10 +80,30 @@ fn init_widget(
     widget.focus();
     widget.blur();
 
+    filter_paste(&widget);
+
     Ok(widget)
 }
 
+fn filter_paste(elt: &HtmlElement) {
+    // Only insert text when pasting something
+    js! {
+        @{&elt}.addEventListener("paste", function (evt) {
+            evt.preventDefault();
+            document.execCommand("insertText", false, evt.clipboardData.getData("text"));
+        });
+    };
+}
+
 pub fn init() -> Result<(), EditorError> {
+    // Check if the user wants to use the basic editor
+    if let Some(x) = window().session_storage().get("basic-editor") {
+        if x == String::from("true") {
+            window().session_storage().remove("basic-editor");
```

Comment:
I think basic-editor should be permanent, and there would then be a button to re-enable the "advanced" editor